### PR TITLE
Update study view layout documentation

### PR DESCRIPTION
**Pull request by zhx828**
# What? Why?
Fix https://github.com/cBioPortal/cbioportal/issues/5145

### CODE REVIEW

File: docs/Study-View.md [hunk 1]
```diff
@@ -15,33 +15,32 @@ For combination chart, as long as one of the clinical attributes has been set to
...
+The layout of a chart is determined mainly based on priority. Higher priority will promote chart closer to the left-top. 
 
-![image](https://user-images.githubusercontent.com/5400599/28329358-8e91312a-6bb7-11e7-97aa-a56de2abf505.png)
+In order to improve the layout, we added a layout algorithm layer. We use pie chart size as the base dimension. 1 grid will be one row * one column in the calculation. Pie chart uses one grid. Bar chart uses two grids. All charts will be placed from left to right, top to bottom. When an even size of the dimension is detected, we will promote the small charts to fit in the space. 
 
-In the example above, `With Mutated Data` and `With CNA Data`  will then be promoted to the empty space (The dashed border boxes in the above image) and the following image shows the expected result.
+##### Before:
```
Comment (jjgao) [inline docs/Study-View.md]: What do Before/After mean?

File: docs/Study-View.md [hunk 2]
```diff
@@ -15,33 +15,32 @@ For combination chart, as long as one of the clinical attributes has been set to
...
+Study view page is fully responsive, it will try to fit more charts in the same height as many as possible base on the browser width.
 
-In order to improve the layout, we added a layout algorithm layer. The algorithm tries to fit all charts into a 2 by 2 matrix (Mutated Genes Table occupies 2 by 2 space). When a chart can not be fitted in the first matrix, the second matrixed will be generated. And the second matrix will have lower priority than the first one. If later chart can fit into the first matrix, then its priority will be promoted.
+The layout of a chart is determined mainly based on priority. Higher priority will promote chart closer to the left-top. 
 
-![image](https://user-images.githubusercontent.com/5400599/28329358-8e91312a-6bb7-11e7-97aa-a56de2abf505.png)
+In order to improve the layout, we added a layout algorithm layer. We use pie chart size as the base dimension. 1 grid will be one row * one column in the calculation. Pie chart uses one grid. Bar chart uses two grids. All charts will be placed from left to right, top to bottom. When an even size of the dimension is detected, we will promote the small charts to fit in the space.
```
Comment (jjgao) [inline docs/Study-View.md]: Could you rephrase "When an even size of the dimension is detected, we will promote the small charts to fit in the space"? Not sure what "even size of the dimension" means.

File: docs/Study-View.md (hunk 1 above)
Comment (zhx828) [reply docs/Study-View.md]: Before/After applying the algorithm

Comment (jjgao) [reply docs/Study-View.md]: Maybe we don't have to show this example and the two images for simplification?
